bodybank/character_generator/views.py

```diff
@@ -52,5 +52,9 @@ def ego_delete(request, pk):
     
     return render(request, 'delete.html', {'ego':ego})
 
+def list_morphs(request):
+    morphs = Morph.objects.all()
+    return render(request, 'morph_list.html',{'morphs':morphs})
```

Comment:
missing space after comma, not urgent now, just a pointer to run flake8 at some point